include/seqan3/range/views/to.hpp

```diff
@@ -7,21 +7,13 @@
 
 /*!\file
  * \author Hannes Hauswedell <hannes.hauswedell AT fu-berlin.de>
- * \brief Provides seqan3::views::to.
+ * \brief [DEPRECATED] Provides seqan3::views::to.
+ * \deprecated This header will be removed in 3.1. Please use seqan3/utility/views/to.hpp instead.
  */
 
 #pragma once
 
-#include <seqan3/std/ranges>
+#include <seqan3/utility/views/to.hpp>
 
-#include <range/v3/range/conversion.hpp>
-
-#include <seqan3/core/platform.hpp>
-
-namespace seqan3::views
-{
-
-//TODO reimplement me
-using ::ranges::to;
-
-} // namespace seqan3::views
+SEQAN3_DEPRECATED_HEADER(
+    "This header is deprecated and will be removed in SeqAn-3.1.0; Please #include <seqan3/utility/views/to.hpp> instead.")
```

Comment:
I don't know if I didn't see it before, or if it came from the changes: newline is missing here.